connect/flow/FlowLogger.hx

```diff
@@ -27,10 +28,13 @@ class FlowLogger {
 
     public function closeFlowSection():Void {
         Env.getLogger().closeSection();
+        Env.getLogger().setFilename(null);
```

Comment:
Can't this two statements (setFilename and setContextRequest) be done in closeRequestSection? That way they are perfectly paired:

openRequestSection -> Set filename and context for request.
closeRequestSection -> Set filename and context for null.